tests/Doctrine/Tests/Mocks/ConnectionMock.php

```diff
@@ -86,12 +97,24 @@ public function lastInsertId($seqName = null)
      */
     public function fetchColumn($statement, array $params = [], $colnum = 0, array $types = [])
     {
+        if (null !== $this->_fetchOneException) {
```

Comment:
This makes sense, but I'll probably have to change it to use the mocking framework to do this (on merge)